sky/backends/backend_utils.py

```diff
@@ -1634,7 +1665,9 @@ def _update_cluster_status_no_lock(
         # set the status to UP, as the `get_node_ips` function uses ray
         # to fetch IPs and starting ray is the final step of sky launch.
         record['status'] = global_user_state.ClusterStatus.UP
-        handle.head_ip = ips[0]
+        handle.head_ip = external_ips[0]
+        handle.stable_cluster_internal_ips = get_stable_cluster_internal_ips(
+            handle)
```

Comment:
`get_node_ips` can be quite time consuming, can we reuse the `external_ips` we get from L1648?